.travis.yml

```diff
@@ -1,17 +1,12 @@
 language: csharp
 sudo: false
-mono:
-  - 4.6.2 # in most common distros
-  - latest
-os:
-  - linux
-  - osx
 matrix:
-  exclude:
-    - os: osx
+  include:
+    - os: linux
+      mono: 4.6.2
+    - os: linux
       mono: latest
-  allow_failures:
-    - mono: latest
-  fast_finish: true
```

Comment:
(Although actually, given all our build is wrapped up in Cake - I guess fast_finish is a no-op anyway.